.idea/modules.xml

```diff
@@ -3,7 +3,9 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/2018-Scouting.iml" filepath="$PROJECT_DIR$/2018-Scouting.iml" />
+      <module fileurl="file://C:\Users\usx06487\StudioProjects\2018-Scouting\2018-Scouting.iml" filepath="C:\Users\usx06487\StudioProjects\2018-Scouting\2018-Scouting.iml" />
```

Comment:
Duplicate module as the line above, but with absolute path.  Should be deleted.  Same with the line below.